.github/workflows/x-create-tags.yml

```diff
@@ -0,0 +1,69 @@
+
+name: X Create tags
+
+on:
+  workflow_call:
+    inputs:
+      gh-org:
+        required: true
+        type: string
+      tag:
+        required: true
+        type: string
+      branch:
+        required: false
+        type: string
+        default: main
+      set-versions:
+        required: false
+        type: boolean
+        default: false
+    secrets:
+      GH_TOKEN:
+        required: true
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  create-tags:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
```

Comment:
I don't think we need to use a matrix here, this is only relevant when we want to run multiple jobs.

Reply:
Fixed in follow-up commit.